sympy/printing/numpy.py

```diff
@@ -63,6 +63,14 @@ def _print_seq(self, seq):
         delimiter=', '
         return '({},)'.format(delimiter.join(self._print(item) for item in seq))
 
+    def _print_Add(self, e):
+        return '{}({})'.format(self._module_format(self._module + '.sum'),
```

Comment:
Doesn't this have the problem with #5642?

Reply:
maybe? would `reduce(add, ...))` make a difference?

Reply:
Not for the particular issue

Reply:
@asmeurer I tried to figure out what a regression test would look like. You don't happen to able to construct one?